src/Controller/CategoryController.php

```diff
@@ -10,4 +10,8 @@ public function index(): string
     {
         return $this->twig->render('Category/index.html.twig');
     }
+    public function add(): string
```

Comment:
Be more precise when you name a function: in this case addCategory() would be better.